compiler/Makefile

```diff
@@ -9,7 +9,7 @@ test:
 	$(BAL) build $(BAL_BUILD_FLAGS)
 
 $(COMPILER_JAR): $(COMPILER_SRC)
-	$(BAL) build $(BAL_BUILD_FLAGS) --skip-tests
+	$(BAL) build $(BAL_BUILD_FLAGS)
```

Comment:
Don't see why this is part of this change.

Reply:
Yes, this could be part of the migration pr. (anyway this pr will have to happen after that).